test/server.test.js

```diff
@@ -90,9 +91,8 @@ describe('RdHandler', function () {
     it('Requests on behalf of the client via external proxy and returns the response even if request by tool fails', function (done) {
       this.timeout = 5000;
       for (var i = 0; i <= constants.MAX_RETRIES; i++) {
-        testHelper.nockGetRequestWithError(constants.HUB_STATUS_URL, 'http');
+        testHelper.nockGetRequestWithError(constants.HUB_STATUS_URL, 'https');
```

Comment:
lets add specs for both the cases, i.e. `http` and `https`, instead of just for https...

Reply:
you can also check for the lines covered by opening the `index.html` in `coverage/` folder of `requests-debugger` post you run tests. And then add tests accordingly...

Reply:
Added

Reply:
can be resolved